governator-jersey/src/main/java/com/netflix/governator/guice/jetty/GovernatorJerseyServletModule.java

```diff
@@ -0,0 +1,55 @@
+package com.netflix.governator.guice.jetty;
+
+import com.google.inject.servlet.ServletModule;
+import com.sun.jersey.guice.JerseyServletModule;
+import com.sun.jersey.guice.spi.container.servlet.GuiceContainer;
+
+/**
+ * Enhancement to JerseyServletModule by allowing the GuiceContainer and Jersey to be configured
+ * via bindings instead of statically in a Module.
+ * 
+ * You can provide the configuration values to Jersey using the binding
+ * 
+ * ```java
+ *  @Provides
+ *  @ServletContainerProperties 
+ *  Map<String, Object> getJerseyProperties() {
```

Comment:
I'll request a binding to ResourceConfig instead of this so we can tie it to archaius.